### DISCUSSION TITLE
🎉 2023-01-28 Bundle Release

### DISCUSSION THREAD

**Discussion by cathay4t** [rust-netlink/.github#4] [Announcements]
## Highlights
 * Using [`#[non_exhaustive]`][1] to preserve backwards comparability.
 * Removed reexports.
 * All crates has their own CHANGELOG file.
 * New crate `netlink-packet-xfrm` for xfrm netlink packet types.

## Versioning
 * netlink_sys 0.8.3 -> 0.8.4
 * netlink-packet-utils 0.5.1 -> 0.5.2
 * netlink-packet-core 0.4.2 -> 0.5.0
 * netlink-package-route 0.13.0 -> 0.15.0
 * netlink-package-audit 0.4.2 -> 0.5.0
 * netlink-proto 0.10.0 -> 0.11.1
 * rtnetlink 0.11.0 -> 0.12.0
 * netlink-packet-generic 0.3.1 -> 0.3.2
 * genetlink 0.2.3 -> 0.2.4
 * ethtool 0.2.3 -> 0.2.4
 * mptcp-pm 0.1.1 -> 0.1.2
 * netlink-packet-sock-diag 0.3.1 -> 0.4.1
 * wl-nl80211 0.1.0 -> 0.1.1
 * netlink-packet-xfrm 0.1.0 -> 0.1.1
 * audit 0.7.1 -> 0.7.2

[1]: https://doc.rust-lang.org/reference/attributes/type_system.html

PS: Some crate like `audit` and `rtnetlink` do not have `non_exhaustive` and reexport enforced yet. Will do it next bundle release.

**Comment by stappersg** [comment]
Because there is #6 do I assume that this bundle release is completed.
